### PR TITLE
isDisabled hook fix

### PR DESCRIPTION
**Pull request by marian-buc**
## What?
Fixes the gql bug by disabling the tracking hook on pages that do not expose a quoteCartId in the url

## Why?

**Ticket(s): []**

### CODE REVIEW

File: src/client/utils/tracking/trackOfferEvent.ts [hunk 1]
```diff
@@ -96,13 +96,14 @@ export const useTrackOfferEvent = () => {
...
   const [selectedInsuranceTypes] = useSelectedInsuranceTypes()
-
   const { data: quoteCartQueryData } = useQuoteCartQuery({
     variables: {
-      id: quoteCartId!,
+      id: quoteCartId || '',
       locale: isoLocale,
     },
+    skip: isDisabled,
   })
```
Comment (mehdielket) [inline src/client/utils/tracking/trackOfferEvent.ts:107]: This query will still be called many times and fail in the landing page and during the embark flow. Can we try to find a solution where we do not call it to start with?

Comment (marian-buc) [reply src/client/utils/tracking/trackOfferEvent.ts:107]: no, it will not be called, that's what skip param does :D

**Review by mehdielket — DISMISSED** [review]
🚀

**Review by dendoesit — COMMENTED** [review]
when following the normal flow the param will be 'ownership' in this case as we still do not have any quoteCartId yet.
image

**Review by dendoesit — CHANGES_REQUESTED** [review]
I believe this can be closed as we have the other fix ticket.

**Pull request closed without merging**